partiql-plan/src/main/java/org/partiql/plan/JoinType.java

```diff
@@ -14,6 +14,22 @@ public class JoinType extends Enum {
     public static final int RIGHT = 3;
     public static final int FULL = 4;
 
+    @NotNull
+    @Override
+    public String name() {
+        switch (code()) {
+            case INNER:
+                return "INNER";
+            case LEFT:
+                return "LEFT";
+            case RIGHT:
+                return "RIGHT";
+            case FULL:
+                return "FULL";
+        }
+        return "UNKNOWN";
```

Comment:
Similar to https://github.com/partiql/partiql-lang-kotlin/pull/1688, do we see uses for an `UNKNOWN` variant yet? If not, perhaps we can remove it for now to be consistent with the AST enums?

Reply:
Also some of the `name` functions follow a different implementation (i.e. some use `String.valueOf(code);` for the default). We should align them to the same implementation

Reply:
Removed them! And, I made the `name()` contract a bit more concrete. See the update.